include/sir/errors.h

```diff
@@ -114,7 +116,7 @@ static const struct {
     {_SIR_E_PLUGINDAT, "Data produced by plugin is invalid"},
     {_SIR_E_PLUGINVER, "Plugin interface version unsupported"},
     {_SIR_E_PLUGINERR, "Plugin reported failure"},
-    {_SIR_E_PLATFORM,  "Platform error code %d: %s"},
+    {_SIR_E_PLATFORM,  _SIR_E_PLATFORM_ERRORFORMAT},
```

Comment:
That actually squashes the warning? What if we change the entry in the map to 'const char* const'?

Reply:
I just tried, no difference.